Dokument/Dateien/Magie_Essenz.tex

```diff
@@ -1,103 +1,870 @@
+\subsubsection{Intuitive Nutzung}
+Da die Weiterleitung von elektrischen Impulsen an Nerven dem Prinzip des Gleichstroms folgt, ist dies bei den Besitzern beschleunigt.
+Daraus folgt eine etwas schnellere körperliche Reaktion auf Situationen.
+Zudem schweben die anwendenden Lebewesen durch einen Blitzeinschlag nicht in Lebensgefahr.  
+
+\subsubsection{Erlernbare Steigerungen}
+\begin{outline}
+	\1 Es kann ein Potentialunterschied zwischen zwei Punkten hergestellt werden, der zu einem Ausgleich der Energie über einen Blitz führt.
+	Die Stärke des Blitzes und die Entfernung der Punkte a) zueinander und b) vom Anwender sind von dessen Fähigkeiten abhängig.
+	Zu Beginn wird nur eine Übertragung zwischen sich und einem anderen Punkt möglich sein.
+	\1 Entflammbare Objekte können angezündet werden, da der Reaktion zwischen Objekt und dem Sauerstoff aus der Luft der nötige Schubs zum Start gegeben wird.
+	\1 Mit der nötigen Energiemenge lässt sich das Wasser in der Luft in Sauerstoff und Wasserstoff spalten und somit Knallgas herstellen.
+	Kommt das Knallgas mit einem Funken in Kontakt, führt das zu einer sofortigen explosiven Revers-Reaktion, bei der sich die beiden wieder zu Wasser verbinden.
+	Einflüsse, Stellschrauben u.ä. für die Erweiterung dieses Skills zur Anwendung im Spiel finden sich \zB auf \href{https://de.wikipedia.org/wiki/Knallgas}{Wikipedia}.
+	\1 ...
+\end{outline}
+
+\subsubsection{Meisterliche Beherrschung} 
+\begin{outline}
+	\1 Der elektrische Signalfluss an Nervenzellen lässt sich stören, womit zum Beispiel eine Reaktion des Gegenübers verhindert wird. 
+	Das ist allerdings recht unpräzise, da die Funktionsweise des Körpers noch nicht so weit bekannt ist. 
+	Durch das Spüren der Ströme konnten allerdings einige Erkenntnisse gewonnen werden. 
+	So lässt sich \zB sämtlicher elektrischer Fluss in einzelne Gliedmaßen ausschalten, wodurch diese nicht mehr bewegt werden können.
+	\1 Ebenso kann man das Gehirn von jemanden lahm legen und ihn damit effektiv töten.
+	\1 Das Herz wird von einem Nervenknoten, dem Sinus, kontrolliert. 
+	Bei Herzstillstand kann man es wieder zum schlagen bringen, indem man diesem Knoten einen elektrischen Impuls gibt, damit das Herz wieder zu arbeiten beginnt (siehe auch \href{https://de.wikipedia.org/wiki/Defibrillator}{Defibrillator}).
+	\1 Schnelleres Denken.
+	\1 ...
+\end{outline}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Gravitation}\label{magie:gravitation}
+Diese Magieform erlaubt eine Beeinflussung der Gravitation.
+
+\subsubsection{Verbreitung}
+Dies ist eine äußerst seltene Magieform, die von keinen bisher bekannten Tieren oder Pflanzen verwendet wird.
+Auch unter den Hominini ist sie nicht verbreitet.
+
+\subsubsection{Bedeutung im Spiel}
+Derzeit keine Bedeutung im Spiel.
+
+\subsubsection{Intuitive Nutzung}
+Intuitiv kann man damit die Auswirkung der Gravitation auf sich selbst reduzieren, was u.a. zu deutlich höheren und weiteren, aber auch unpräziseren Sprüngen führt.
+
```

Comment:
oder auch das eigene Gewicht erhöhen, um z.B. in einem Sturm standfest zu bleiben

Reply:
nice